coderd/workspaceagents.go

```diff
@@ -1463,6 +1473,96 @@ func (api *API) workspaceAgentsExternalAuthListen(ctx context.Context, rw http.R
 	}
 }
 
+// @Summary User-scoped agent coordination
+// @ID user-scoped-agent-coordination
+// @Security CoderSessionToken
+// @Tags Agents
+// @Success 101
+// @Router /tailnet [get]
+func (api *API) tailnet(rw http.ResponseWriter, r *http.Request) {
```

Comment:
```suggestion
func (api *API) tailnetRPCConn(rw http.ResponseWriter, r *http.Request) {
```
tailnet is slightly too generic for this huge `API` object